sdk/appconfiguration/app-configuration/test/public/utils/testHelpers.ts

```diff
@@ -30,67 +29,36 @@ export interface CredsAndEndpoint {
 export async function startRecorder(that: Mocha.Context): Promise<Recorder> {
   const recorderStartOptions: RecorderStartOptions = {
     envSetupForPlayback: {
-      APPCONFIG_CONNECTION_STRING:
-        "Endpoint=https://myappconfig.azconfig.io;Id=123456;Secret=123456",
       AZ_CONFIG_ENDPOINT: "https://myappconfig.azconfig.io",
-      AZURE_CLIENT_ID: "azure_client_id",
-      AZURE_CLIENT_SECRET: "azure_client_secret",
-      AZURE_TENANT_ID: "azuretenantid",
-    },
-    sanitizerOptions: {
-      connectionStringSanitizers: [
-        {
-          fakeConnString: "Endpoint=https://myappconfig.azconfig.io;Id=123456;Secret=123456",
-          actualConnString: env.APPCONFIG_CONNECTION_STRING,
-        },
-      ],
     },
+    removeCentralSanitizers: [
+      "AZSDK3447", // .key in the body is not a secret for key-value App Config pair
+      "AZSDK3490", // etag value in If-Match header is not a secret and is needed for etag test
+      "AZSDK2030", // operation-location header is not a secret and is needed for long running operation tests
+      "AZSDK3493", // .name in the body is not a secret
+    ],
   };
 
   const recorder = new Recorder(that.currentTest);
   await recorder.start(recorderStartOptions);
   return recorder;
 }
 
-export function getTokenAuthenticationCredential(): CredsAndEndpoint {
-  const requiredEnvironmentVariables = [
-    "AZ_CONFIG_ENDPOINT",
-    "AZURE_CLIENT_ID",
-    "AZURE_TENANT_ID",
-    "AZURE_CLIENT_SECRET",
-  ];
-
-  for (const name of requiredEnvironmentVariables) {
-    const value = env[name];
-
-    if (value == null) {
-      if (tokenCredentialsNotPresentWarning) {
-        tokenCredentialsNotPresentWarning = true;
-      }
-
-      throw new Error("Invalid value for requiredEnvironmentVariables");
-    }
-  }
-
-  return {
-    credential: createTestCredential(),
-    endpoint: env["AZ_CONFIG_ENDPOINT"]!,
-  };
-}
-
 export function createAppConfigurationClientForTests(
-  options?: AppConfigurationClientOptions,
+  options?: AppConfigurationClientOptions & {
+    testCredential?: TokenCredential;
+  },
 ): AppConfigurationClient {
-  const connectionString = env["APPCONFIG_CONNECTION_STRING"];
```

Comment:
Can we remove the connection string environment variable from [`test-resources.json`](https://github.com/Azure/azure-sdk-for-js/blob/fa75f392f6090af0cf678b14c3fd5573af96f048/sdk/appconfiguration/test-resources.json#L86-L89)? We're not using it anywhere anymore, right?

Reply:
Yes, I removed both connection string & client secret too. I'm leaving tenant ID and client ID in the `test-resources.json` file, but we only need the endpoint for the tests. Any thoughts on removing them?

Reply:
The live tests pass with client secret removed [here](https://dev.azure.com/azure-sdk/internal/_build/results?buildId=3829399&view=results). The failure is with the samples so I updated all to use DAC